docker/bookkeeper/Dockerfile

```diff
@@ -16,23 +16,25 @@ RUN apt-get update \
 &&  mkdir -p /opt \
 &&  cd /opt
 
-RUN wget http://www.apache.org/dist/bookkeeper/bookkeeper-4.4.0/bookkeeper-server-4.4.0-bin.tar.gz \
-&& tar xvzf  bookkeeper-server-4.4.0-bin.tar.gz \
-&& mkdir -p /opt/bk_all \
-&& mv bookkeeper-server-4.4.0/ /opt/bk_all/
+ENV BOOKKEEPER_VERSION=4.5.0 \
+    ZOOKEEPER_VERSION=3.5.3-beta
```

Comment:
io.pravega.test.system.framework.services.ZookeeperService#ZK_IMAGE points to zookeeper:3.5.2-alpha it should also be updated.

Reply:
That seems like it's out of scope for this PR. This used to use 3.5.1-alpha, so they never aligned anyway.